ignite/pkg/cosmosgen/install.go

```diff
@@ -32,7 +34,6 @@ func InstallDependencies(ctx context.Context, appPath string) error {
 			cmdrunner.DefaultWorkdir(appPath),
 		).
 		Run(ctx,
-			step.New(step.Exec("go", append([]string{"get"}, plugins...)...)),
```

Comment:
As explained here [0], when you use the `tools.go` pattern, you don't need to run `go get`, only `go install`.  

This change is only motivated by the fact that `go get` has the bad side effect to downgrade the sdk to v0.44.

You can observe this behavior by running `go get github.com/cosmos/gogoproto/protoc-gen-gocosmos` in a chain for instance.

[0] https://github.com/go-modules-by-example/index/blob/master/010_tools/README.md